docs/community/contributing.rst

```diff
@@ -84,7 +84,19 @@ Now you have a development installation of Parcels, as well as a bunch of develo
 Pixi workflows
 ~~~~~~~~~~~~~~
 
-You can use the following Pixi commands to run common development tasks.
+**Typical development workflow**
+
+1. Make your code changes
```

Comment:
Maybe indicate here how to activate the pixi environment to change the code, to keep it clear (e.g, ```pixi shell -e test-latest```).

Reply:
Yes! I had this but I forgot to commit it :/

Reply:
I've added a part now in the different environments sections

<img width="631" height="753" alt="image" src="https://github.com/user-attachments/assets/33768992-ab06-49db-95ba-dec9b4b5d846" />

Reply:
That's perfect!